### PR TITLE
We don't want those Python packages

### PR DESCRIPTION
**Pull request by hroncok**

### CODE REVIEW

**Comment by hroncok** [comment]
Based on the content of the list, I've added "binary" RPM names, not component names. Is that correct?

File: configs/view-eln.yaml [hunk 1]
```diff
@@ -75,6 +75,15 @@ data:
   # be pulled into a RHEL-like environment.
   - python3.6
 
+  # Miro Hrončok:
+  # The older Python version might accidentally be pulled in by not yet rebuilt packages
+  - python3.8
```
Comment (hroncok) [inline configs/view-eln.yaml:80]: As a side note, could you please rebuild glade-libs and unbound-libs with Python 3.9?

**Comment by torsava** [comment]
+1

File: configs/view-eln.yaml (hunk 1 above)
Comment (sgallagher) [reply configs/view-eln.yaml:80]: > As a side note, could you please rebuild glade-libs and unbound-libs with Python 3.9?

I will take care of this.

**Pull request merged**